.circleci/config.yml

```diff
@@ -46,12 +46,13 @@ jobs:
       - checkout
       - run:
           name: "Build Sonobuoy binaries"
-          command: make build/linux/amd64/sonobuoy build/linux/arm64/sonobuoy
+          command: make build/linux/amd64/sonobuoy build/linux/arm64/sonobuoy build/linux/mips64le/sonobuoy
       - persist_to_workspace:
           root: ~/project
           paths: 
             - build/linux/amd64/*
             - build/linux/arm64/*
+            - build/linux/mips64le/* 
```

Comment:
I don't think we should publish any images that we don't necessarily test / support. I'm okay with adding the `Makefile` entry for folks to build their own image, but I'm not sure if we should publish this.